timescaledb/overview/release-notes/index.md

```diff
@@ -21,6 +21,22 @@ Want to stay up-to-date with new releases? Subscribe to get
 * Introduce fixed schedules for background jobs and the ability to check job errors.
 * Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.
 
+## 2.9.3 (2023-02-06)
+
+This release contains a security fix(#5259) and multiple bug fixes since the 2.9.2 release.
```

Comment:
```suggestion
This release contains bug fixes since the 2.9.2 release and a fix for a security vulnerability (#5259). You can check the [security advisory](https://github.com/timescale/timescaledb/security/advisories/GHSA-44jh-j22r-33wq) for more information on the vulnerability and the platforms that are affected.

```